python-threatexchange/threatexchange/fb_threatexchange/api.py

```diff
@@ -21,6 +21,47 @@
 from .api_representations import ThreatPrivacyGroup
 
 
+def get_app_token(cli_option: str = None) -> str:
```

Comment:
TODO: Move this to storage / collabConfig storage.

Reply:
I think it has a place in cli main.py